hexchat/snapcraft.yaml

```diff
@@ -1,6 +1,6 @@
 name: hexchat
-version: "2.12.1"
-summary: IRC client for X based on X-Chat 2
+version: 2.12.1
```

Comment:
We have as best practice to enforce version being in a string field. (that prevents if we update to "2.20" to be translated in "2.2". Please revert that change.